lib/gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -102,12 +103,14 @@ def initialize_logfile_disk
     end
 
     def start_evm
+      return true unless evm_was_running
```

Comment:
@carbonin Thank you for the review. I had thought of doing it the way you suggested but then thought that would require the caller to have knowledge of what the methods `start_evm` and `stop_evm` are doing and how. So I think it would be better to leave it as is. Let's discuss if you feel strongly about it.